HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -270,5 +281,6 @@ def customizeHLTforCMSSW(process, menuType="GRun"):
     # process = customiseFor12718(process)
 
     process = checkHLTfor43774(process)
+    process = customizeHLTfor46062(process)
```

Comment:
This is problematic if it needs to go into a 14_0 release for 2024 data taking - given that HLT development has moved to 14_1 already. So can this be deferred until P5/T0 has moved to 14_1?

Reply:
It may be used for MC production and not dor data taking